app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/attachments/attachments.jsx

```diff
@@ -22,56 +26,134 @@ const cx = classNames.bind(styles);
   (state) => ({
     attachments: attachmentItemsSelector(state),
     loading: attachmentsLoadingSelector(state),
+    pagination: attachmentsPaginationSelector(state),
   }),
   {
-    fetchAttachmentsAction,
+    fetchAttachmentsConcatAction,
     openAttachmentAction,
   },
 )
 @injectIntl
-export class Attachments extends React.Component {
+export class Attachments extends Component {
   static propTypes = {
     intl: intlShape.isRequired,
     attachments: PropTypes.array,
     activeItemId: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+    pagination: PropTypes.object,
     loading: PropTypes.bool,
     onChangeActiveItem: PropTypes.func,
-    fetchAttachmentsAction: PropTypes.func,
+    fetchAttachmentsConcatAction: PropTypes.func,
     openAttachmentAction: PropTypes.func,
+    isMobileView: PropTypes.bool,
   };
 
   static defaultProps = {
     attachments: [],
+    pagination: {},
     loading: false,
     activeItemId: null,
     onChangeActiveItem: () => {},
-    fetchAttachmentsAction: () => {},
+    fetchAttachmentsConcatAction: () => {},
     openAttachmentAction: () => {},
+    isMobileView: false,
   };
 
-  state = {
-    mainAreaVisible: this.props.activeItemId !== null,
-  };
+  static getVisibleThumbs = (isMobile) =>
+    isMobile ? MOBILE_VISIBLE_THUMBS : DEFAULT_VISIBLE_THUMBS;
+
+  constructor(props) {
+    super(props);
+    const { activeItemId, isMobileView, pagination } = props;
+
+    const visibleThumbs = Attachments.getVisibleThumbs(isMobileView);
+    const currentThumb = activeItemId
+      ? Math.floor(activeItemId / visibleThumbs) * visibleThumbs
+      : 0;
+
+    this.state = {
+      mainAreaVisible: activeItemId !== null,
+      activeItemId,
```

Comment:
It looks a little bit strange, you receive `activeItemId` as a prop, and then change it inside this component (update its value in the state) and not update `activeItemId` in the outer component. Is this behavior correct?

Reply:
I update it in the outer component also. F.e. looks on [this](https://github.com/reportportal/service-ui/pull/1300/files/467eaa13d0692d965e2341b47021bbc8f44ecfa1#diff-48d4e51950475e052cf1f5f5eef6f703R103) line

Reply:
So, in this case, my do you need to save it in the state? Maybe you can just use it directly from props?

Reply:
It was a problem with the active tab content rendering process in `infoTabs`.
Now fixed and `activeItemId` uses directly from props.